src/cpp/transport/TCPTransportInterface.cpp

```diff
@@ -1142,6 +1151,10 @@ void TCPTransportInterface::SocketConnected(Locator_t locator, const asio::error
         }
         else
         {
+            if (error.value() == 111)
```

Comment:
Please use a constant for this. I think it corresponds to asio::error::basic_errors::connection_refused

Reply:
Changed.